packages/validator/README.md

```diff
@@ -0,0 +1,98 @@
+# `@airnode/validator`
+
+Package for json specifications validation. Validation process can be configured with validator templates.
+
+## Usage
+
+The validator can be run as an npm script, by providing template of specification, and the path to the JSON file of specification that will be validated:
+```sh
+npm run validate --template="[template]" --specs="[specsFile]"
+```
+
+Templates are case-insensitive, valid templates are: `config`, `OIS`, `apiSpecifications`/`apiSpecs` and `endpoints`. If arguments are in order `template`, `specs` command can be simplified: `npm run validate [template] [specsFile]`:
+```sh
+npm run validate config exampleSpecs/config.specs.json
+```
+
+Validator will automatically validate the latest available version of provided template, in case specific version should be used in validation, it can be passed as `--templateVersion` argument or as the last argument if they are ordered:
```

Comment:
Thats quite good suggestion, the versioning needs a bit of work though, it will be probably modified on [this branch](https://github.com/api3dao/airnode/tree/validator-templates), where I will try to include this.